src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -7,6 +7,7 @@
 		<lending-category-section
 			title="Support Borrowers from Fundación Pro Mujer"
 			:loans="mfiRecommendationsLoans"
+			enable-loan-card-exp="enableLoanCardExp"
```

Comment:
This needs to be `:enable-loan-card-exp`. There's currently an error in the console for this.

Reply:
fixed, ty